src/PerfView/SupportFiles/Tutorial.cs.txt

```diff
@@ -0,0 +1,64 @@
+using System;
```

Comment:
Please remove this file from the change.  It gets packaged from a difference source location during the build process.